### ISSUE TITLE
DisableLinterReason needs to be whitespace/linebreak-aware

### ISSUE THREAD

**Issue by Undistraction** [brigade/scss-lint#636]
The following fails to trigger the DisableLinterReason linter.

```
// Some Sectioning Comment
// ---------------------------------------------

// scss-lint:disable DeclarationOrder
.Example {}
```

Note the format of the preceding comment is irrelevant. The point is that this comment isn't explaining the reason behind using the linter comment, and this should be clear given the whitespace/linebreak.

*scss-lint 0.42.2*

**Comment by sonalkr132** [comment]
Check [disable_linter_reason_spec.rb](https://github.com/brigade/scss-lint/blob/50dbcaf0119c1edde250c292c5717551f725fb23/spec/scss_lint/linter/disable_linter_reason_spec.rb#L38-L49)

```Ruby
  context 'when a reason precedes a disabling comment, at a distance' do
    let(:scss) { <<-SCSS }
      // We like using `border: none` in our CSS.

      // scss-lint:disable BorderZero
      p {
        margin: 0;
      }
    SCSS

    it { should_not report_lint }
  end
```

It seems that it is valid to give *explanation* separated by line break.

**Comment by sds** [comment]
Thanks for pointing out that spec, @sonalkr132.

Yes, it is allowed to separate the reason with a blank line.

**Comment by Undistraction** [comment]
FWIW I think this is not desirable behaviour. It would make more sense to assume the comment is not broken, otherwise any unrelated comment will case the linter to pass it.
